README.md

```diff
@@ -59,5 +58,5 @@ bidrag. Ta en titt på [CONTRIBUTING.md](CONTRIBUTING.md) for hvordan du går fr
 
 ## Licenses
 
-* Source code is licensed under MIT
-* The MuseoSans fonts are licensed separately. See LICENSE-fonts.md.
+-   Source code is licensed under MIT
```

Comment:
Not sure if this is Prettier running wild, but this change breaks the markdown generated bullet list. The reformatting of this file probably shouldn't be included in the final commit.

Reply:
It is, I will fix this once I squash the commits and fix the changes to files that shouldn't have been included :)